taichi/gui/gui.h

```diff
@@ -805,14 +815,12 @@ class GUI : public GUIBase {
 
   void update() {
     frame_id++;
-    redraw_widgets();
```

Comment:
Why remove?

Reply:
@archibate I moved it to a separate function called `update_gui` to make it cleaner. [Here](https://github.com/taichi-dev/taichi/pull/1817/files#diff-bfe811ffeab1312fe18395d9aa2a79beR847-R853), I do

```
if (show_gui) {
    update_gui();
}
```